app/models/agents/user_location_agent.rb

```diff
@@ -68,7 +73,7 @@ def receive_web_request(params, method, format)
     def handle_payload(payload)
       location = Location.new(payload)
 
-      if location.present?
+      if (location.present? && (payload["accuracy"] < interpolated[:max_accuracy]))
```

Comment:
:wink:

Depends if @stvnrlly prefers a missing accuracy to be treated as 100% accurate or 0% accurate.